bin/journey-test.sh

```diff
@@ -233,15 +233,15 @@ solve_all_exercises() {
 
     pushd ${exercism_exercises_dir}/${TRACK}/${exercise}
     # Check that tests compile before we strip @Ignore annotations
-    "$EXECPATH"/gradlew compileTestJava
+    "$EXECPATH"/gradlew compileTestJava --configure-on-demand --parallel
```

Comment:
As far as I can tell, this step being run in parallel does not imply all the journey tests are run in parallel; see the console output of the journey tests here: https://travis-ci.org/exercism/java/jobs/251297234